### ISSUE TITLE
requirement for the hardware system

### ISSUE THREAD

**Issue by atefeh-f** [DeepLearning4BioSeqText/Paper16-DeepChrome#5]
Hi 
       I want to run your code but I dont know about minimum requirement for the hardware system!
       My computer,s  cpu is cori 5 and  includes 4G RAM. When I am runing DeepChrome  I face this error message
       Segmentation fault (core dumped)................] ETA: 0ms | Step: 0ms 
       does DeepChrome need to GPU?
       Would you please help me to solve this error?
       Best Regards

**Comment by rs3zz** [comment]
Hello, 

DeepChrome does not require a GPU to run and can run on your personal CPU machine, given that you have enough memory to handle the datasets. 

What is the size of the dataset that you are running it on? Have you tried running it on smaller datasets on the same machine?

**Comment by atefeh-f** [comment]
Thanks for your quick reply.
I used from REMC dataset. I redused the size of data even to 100 rows but again received this Error message. But this error vanished when I used a system with higher RAM.

I had though if Torch's liberary is not installed correctly then I should not be able to enter it. Beside that I think it has not installed completely, but I could enter it. Nevertheles with reinstalling it in higher RAM system, this problem has removed completely and it is probable that this matter(complete installation) is the reason of it's removing  .
